gui/src/components/mainInput/getSuggestion.ts

```diff
@@ -21,37 +21,48 @@ function getSuggestion(
       let popup;
 
       const onExit = () => {
-        popup?.[0]?.destroy();
-        component?.destroy();
-        onClose();
+        requestAnimationFrame(() => {
+          popup?.[0]?.destroy();
+          component?.destroy();
+          onClose();
+        });
       };
 
       return {
         onStart: (props) => {
-          component = new ReactRenderer(MentionList, {
-            props: { ...props, enterSubmenu, onClose: onExit },
-            editor: props.editor,
+          requestAnimationFrame(() => {
+            component = new ReactRenderer(MentionList, {
+              props: { 
+                ...props, 
```

Comment:
whats this changes, mmm, i never saw tippy tooltip, where does this show?

Reply:
Hmmm

Reply:
It looks like this maybe a refactoring change (not on purpose).  If you look though, it was there previously.  Scroll down and look at the diff.